src/ServicePulse.Host/GlobalSuppressions.cs

```diff
@@ -6,3 +6,4 @@
 using System.Diagnostics.CodeAnalysis;
 
 [assembly: SuppressMessage("StyleCop.CSharp.DocumentationRules", "SA1600:Elements should be documented", Justification = "Not a public API")]
+[assembly: SuppressMessage("Reliability", "CA2007:Consider calling ConfigureAwait on the awaited task", Justification = "console app")]
```

Comment:
Even if it is console app we never suppress configureawait in any production code and I see no reason to deveat from that. Even more so because middleware pieces like this tend to get borrowed into other code pieces

Reply:
Fair enough, I'll fix.